packages/react-components/src/components/RichTextEditor/RichTextSendBoxErrors.tsx

```diff
@@ -80,19 +80,28 @@ export const RichTextSendBoxErrors = (props: RichTextSendBoxErrorsProps): JSX.El
   const onDismiss = useCallback(() => {
     if (systemMessage && !isMessageEmpty(systemMessage)) {
       setSendBoxError({ message: systemMessage, timestamp: Date.now() });
+      return;
     }
-  }, [systemMessage]);
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    if (attachmentProgressError || attachmentUploadsPendingError) {
+      setSendBoxError(undefined);
```

Comment:
Is it to handle situation when 1 error with text `Test` is shown and dismissed and another error with the same text is shown after that?

Reply:
Do we need to set it to `undefined` for all errors, not only attachmentProgressError and attachmentUploadsPendingError?

Reply:
Yes I think it would be safe to dismiss for all errors except system error. Actually I feel we shouldn't add the dismiss button if it's a system error. Will make the change accordingly.